gprt/CMakeLists.txt

```diff
@@ -75,4 +76,6 @@ target_include_directories(gprt
     ${CMAKE_CURRENT_LIST_DIR}
 )
 
+target_compile_features(gprt PRIVATE cxx_std_17)
```

Comment:
Let's make a note in the readme that the C++17 standard is required for compilation just in case someone is using an old compiler.

Reply:
Done!